packages/suite/src/utils/wallet/coinmarket/__tests__/savingsUtils.test.ts

```diff
@@ -6,7 +6,7 @@ import {
     getPaymentFrequencyOptions,
     getStatusMessage,
 } from '../savingsUtils';
-import { FIAT_RATES, SELECTED_PROVIDER } from '../__fixtures__/savingsUtils';
+import { SELECTED_PROVIDER } from '../__fixtures__/savingsUtils';
```

Comment:
I would define something like 

`const USD_BITCOIN_PRICE = 54321` and use it so that it is clear that the last number is price

Reply:
Will also do it for fiat amount

Reply:
57f2822a8eb60dbacd2924812cfb0c0a06da3760